src/redux/Category/categoriesSlice.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+};
+
+const categoriesSlice = createSlice({
+  name: 'categorie',
+  initialState,
+  reducers: {
+    extraReducer: (state) => {
+      state.categories = true;
```

Comment:
- [ ] Nice work here, but I think you should return the expected content which we are expecting from the payload. You can by replacing the true with `action.payload === "Under construction"
          ? "Under construction"
          : state.categories
`

Remember to pass the extraReducer a second parameter of the expected action. You can just use action because that is what we want to do.